contracts/src/lib/CrossMsgHelper.sol

```diff
@@ -172,42 +171,13 @@ library CrossMsgHelper {
             return
                 supplySource.performCall(
                     payable(recipient),
-                    abi.encodeCall(IfaceIpcContract.IpcEntrypoint, (crossMsg)),
+                    abi.encodeCall(IpcHandler.handleIpcMessage, (crossMsg)),
                     crossMsg.value
                 );
         }
         return (false, EMPTY_BYTES);
     }
 
-    // get underlying IpcMsg from crossMsg
-    function getIpcMsg(IpcEnvelope calldata crossMsg) public pure returns (IpcMsg memory ret) {
-        if (isEmpty(crossMsg)) {
-            return ret;
-        }
-        if (crossMsg.kind == IpcMsgKind.Call || crossMsg.kind == IpcMsgKind.Transfer) {
-            IpcMsg memory message = abi.decode(crossMsg.message, (IpcMsg));
-            return message;
-        }
-
-        // return empty IpcMsg otherwise
-        return ret;
-    }
-
-    // set underlying IpcMsg from crossMsg.
-    // This is a pure function, so the argument is not mutated
-    function setIpcMsg(
-        IpcEnvelope memory crossMsg,
-        IpcMsg memory message
-    ) public pure returns (IpcEnvelope memory ret) {
-        if (crossMsg.kind == IpcMsgKind.Call || crossMsg.kind == IpcMsgKind.Transfer) {
-            crossMsg.message = abi.encode(message);
-            return crossMsg;
-        }
-
-        // Cannot set IPCMsg for the wrong kind
-        revert InvalidCrossMsgKind();
-    }
-
```

Comment:
These methods are only used in tests, so I moved them to the tests. The lesser unused code we have in production contracts, the lesser the attack surface.